### PR TITLE
Fixes broken derived sets

### PR DESCRIPTION
**Pull request by andricDu**
Derived sets require a different request body with no filters or size parameters. 

params2JSON can take a boolean to exclude these for derived sets.

### CODE REVIEW

File: dcc-portal-ui/app/scripts/sets/js/services.js [hunk 1]
```diff
@@ -106,13 +106,17 @@
     var _service = this;
 
     // For application/json format
-    function params2JSON(type, params) {
+    function params2JSON(type, params, derived = false) {
```
Comment (btiernay) [inline dcc-portal-ui/app/scripts/sets/js/services.js]: Can you do that in js?

Comment (btiernay) [inline dcc-portal-ui/app/scripts/sets/js/services.js]: Are you sure that isn't assigning to a global?

Comment (andricDu) [inline dcc-portal-ui/app/scripts/sets/js/services.js]: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Default_parameters

Comment (btiernay) [inline dcc-portal-ui/app/scripts/sets/js/services.js]: Browser compat is pretty bad. I would avoid it.

**Pull request merged**